yt_dlp/extractor/common.py

```diff
@@ -151,6 +151,11 @@ class InfoExtractor:
                                    for HDS - URL of the F4M manifest,
                                    for DASH - URL of the MPD manifest,
                                    for MSS - URL of the ISM manifest.
+                    * manifest_data
+                                The media playlist m3u8 data as a string,
+                                only if needed due to the data fetched from the
+                                manifest URL needing to be modified by the extractor.
+                                Only applicable to the native HLS downloader
```

Comment:
This is incorrect, I guess. It really only doesn't apply to ffmpeg
```suggestion
                                Only applicable for HLS formats, does not apply to ffmpeg
```

Reply:
```suggestion
                                Does not apply if ffmpeg is used as downloader
```